monkey/infection_monkey/post_breach/post_breach_handler.py

```diff
@@ -0,0 +1,66 @@
+import logging
+import infection_monkey.config
+import platform
+from file_execution import FileExecution
+from pba import PBA
+
+LOG = logging.getLogger(__name__)
+
+__author__ = 'VakarisZ'
+
+
+# Class that handles post breach action execution
+class PostBreach(object):
+    def __init__(self):
+        self.os_is_linux = False if platform.system() == 'Windows' else True
+        self.pba_list = self.config_to_pba_list(infection_monkey.config.WormConfiguration)
+
+    def execute(self):
+        for pba in self.pba_list:
+            pba.run(self.os_is_linux)
+        LOG.info("Post breach actions executed")
+
+    @staticmethod
+    def config_to_pba_list(config):
```

Comment:
Maybe this should be remade to return a generator?

Reply:
No need, since creating the PBA objects is easy, and all of them are used immediately.